pybricksdev/connections/pybricks.py

```diff
@@ -837,10 +837,14 @@ async def write_gatt_char(self, uuid: str, data, response: bool) -> None:
             raise ValueError("Response is required for USB")
 
         self._ep_out.write(bytes([PybricksUsbOutEpMessageType.COMMAND]) + data)
-        # FIXME: This needs to race with hub disconnect, and could also use a
-        # timeout, otherwise it blocks forever. Pyusb doesn't currently seem to
-        # have any disconnect callback.
-        reply = await self._response_queue.get()
+
+        try:
+            reply = await asyncio.wait_for(
+                self.race_disconnect(self._response_queue.get()),
```

Comment:
`race_disconnect()` doesn't work for USB because it doesn't seem to be technically possible to get a disconnect event from pyusb. That was the point of the comment. So until that is fixed, I think we should leave a FIXME comment to make it clear that we don't expect this to actually work.

Reply:
Thanks for the info -- I wasn't aware of that. Added a FIXME.